modules/@apostrophecms/image/ui/apos/components/AposMediaUploader.vue

```diff
@@ -30,6 +34,7 @@
       :accept="accept"
       multiple="true"
       :disabled="disabled"
+      tabindex="-1"
```

Comment:
the file input is hidden, so we make the drag&drop element focusable and fake-trigger the input file click when pressing space or enter on the drag&drop element,